woocommerce/woocommerce
# Adding test name block with support for entity data store

### All Submissions:






### Changes proposed in this Pull Request:

Adding a basic name block to the product editor, with support for the entity data store.



Closes #37007 .



### How to test the changes in this Pull Request:

1. Checkout branch.
2. Enable `new-product-management-experience` feature flag.
3. Navigate to Products -> Add new.
4. You should see the block editor, with the name block present.
5. If you have redux dev tools, open them and select the `core` store (for some reason this is listed twice for me, and I need to select the 2nd one)
6. Notice when you type into the name field that the `EDIT_ENTITY_RECORD` action is fired.



### Other information:




### FOR PR REVIEWER ONLY:
